client/blocks/reader-related-card-v2/index.jsx

```diff
@@ -94,18 +95,21 @@ export function RelatedPostCard( { post, site, onPostClick = noop, onSiteClick =
 	const classes = classnames( 'reader-related-card-v2', {
 		'has-thumbnail': !! featuredImage
 	} );
+  // TODO what is the difference between post.URL and getPostUrl( post )
```

Comment:
`post.URL` gives the external link to the post on the origin site.

`getPostUrl()` gives the URL to the post within Reader, e.g. /read/feeds/123/posts/123. `getReaderPostUrl()` might have been a better name for it :)